testsuite/tests/parallel/catch_break.ml

```diff
@@ -0,0 +1,63 @@
+(* TEST
+
+* hassysthreads
+include systhreads
+** not-windows
+*** bytecode
+*** native
```

Comment:
I see that the format has changed for some reason. Can someone please point me to the new way of writing this?

Reply:
Yes, see https://github.com/ocaml/ocaml/blob/24ec1395087c1ed41a233b2e36203725b490a117/testsuite/tests/parallel/domain_serial_spawn_burn.ml for an example. I would try to port your test as follows:

```ocaml
(* TEST
hassysthreads;
include systhreads;
not-windows;
{
  bytecode;
}{
  native
}
*)
```

Reply:
Thanks! (For people learning the new format with me: @gasche's solution only misses the semi-column after `native`)